src/modifiers/handleInlineStyle.js

```diff
@@ -14,6 +14,10 @@ const handleChange = (editorState, line, whitelist) => {
         do {
           matchArr = re.exec(line);
           if (matchArr) {
+            if (matchArr[0][0] === " ") {
+              matchArr[0] = matchArr[0].trimLeft();
```

Comment:
`trimLeft` (which is an alias of `trimStart`) has horrendous browser support, can we do this manually?